src/GrafanaDashboardDeploy/src/Program.cs

```diff
@@ -0,0 +1,96 @@
+using Microsoft.Extensions.Logging;
+using Mono.Options;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace DotNet.Grafana
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.SetMinimumLevel(LogLevel.Trace);
+                builder.AddConsole();
+            });
+            ILogger logger = loggerFactory.CreateLogger<Program>();
+
+            var httpClient = new HttpClient();
+            var grafanaClient = new GrafanaClient(loggerFactory.CreateLogger<GrafanaClient>(), httpClient, "http://localhost:3000");
```

Comment:
localhost probably isn't right. :-) This almost certainly needs to be a parameter or configuration setting or something.

Reply:
The individual commands accept the URI and overwrite this property. I wanted to guarantee that the class was created and had a safe default.